### PR TITLE
Add functionality so that a user can book a room

### PR DESCRIPTION
**Pull request by LeahWinters**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Styling- Booking Confirmation Display

### Detailed Description

Added functionality so that when a user is logged in, they can select a room to book and that booking to posted to the booking data set. Also created a confirmation message so that the user knows they have successfully booked a trip.

### Why is this change required? What problem does it solve?

So that a user can book a trip.

### Were there any challenges that arose while implementing this feature? If so, how were the addressed?

My room number was being stored as a string so it was causing issues and not allowing the post to go through. Changed the strictly equals to a loosely equals and that solved the problem.

### Files modified:
- [ ] index.html
- [X] styles.scss
- [X] index.js
- [ ] Other files:

### CODE REVIEW

**Comment by LeahWinters** [comment]
@rachael-t for review
@hannahhch @thatpamiam

File: src/index.js [hunk 1]
```diff
@@ -244,20 +245,66 @@ const displayFilteredAvailableBookings = (user, roomType, date) => {
...
       </section>`)
     });
   }
 }
 
+const selectRoomToBook = (event) => {
+  console.log('event', event.target.id)
+  let roomId = event.target.id;
+  console.log('roomId', roomId);
+  let roomMatch = hotel.allRooms.find(room => room.number == roomId);
```
Comment (rachael-t) [inline src/index.js:259]: Is this where you had to add the loosely equals? If so, you could try refactoring line 257 to parseInt() the string and return it as a string, that way line 259 can be strictly equals and be comparing a number to a number. So line 257 could look like: `let roomId = parseInt(event.target.id);`

**Comment by rachael-t** [comment]
Great job getting the post request working!

File: src/index.js (hunk 1 above)
Comment (LeahWinters) [reply src/index.js:259]: @rachael-t Hey thank you! Yes, that is exactly where I was having the issue. I originally had it written with a parsInt around it, but I was getting a lot of errors with it for some reason. I will try it again though and see if it changes its mind. Thank you for the feed back!

**Pull request merged**